package.xml

```diff
@@ -17,6 +17,7 @@
   <build_depend>libconsole-bridge-dev</build_depend>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros</buildtool_depend>
```

Comment:
I think you should replace `ament_cmake` with `ament_cmake_ros` rather than add it, just based on what's done in other packages already, e.g.:

https://github.com/ros2/rcl/blob/f90c01c19212cf5d8a0dd4fdeb966c5a9910518e/rcl/package.xml#L12

Reply:
As long as the CMake code explicitly uses `ament_cmake` (https://github.com/ros/class_loader/pull/163/files#diff-af3b638bc2a3e6c650974192a53c7291R16) I would rather keep the dependency. Similar to C/C++ code where you should include and depend on all symbols explicitly which are used directly.